frontend/app/MakeRoom/page.tsx

```diff
@@ -0,0 +1,74 @@
+'use client'
```

Comment:
**suggestion:** I'm thinking we could put this file in a rooms folder 

like `/app/rooms/create/page.tsx`

just so all our room related code is organized

would need to change the `<Link>` on the home page if we go this route.

thoughts?